conf/janus.eventhandler.rabbitmqevh.jcfg.sample

```diff
@@ -20,6 +20,7 @@ general: {
 	#exchange = "janus-exchange"
 	route_key = "janus-events"		# Name of the queue for event messages
 	#exchange_type = "fanout" 		# Rabbitmq exchange_type can be one of the available types: direct, topic, headers and fanout (fanout by defualt).
+	#heartbeat = 60 				# Defines after what period of time the TCP connection should be closed if no keepalive is received.
```

Comment:
The comment should say what the unit is: I assume it's the "period of time" in seconds but it's not clear.

Reply:
It makes sense. Description changed to:
Defines the seconds without communication that should pass before considering the TCP connection has unreachable.